app/Actions/AlbumAuthorisationProvider.php

```diff
@@ -0,0 +1,553 @@
+<?php
```

Comment:
By the nature of this Class. I would not place it in the Actions folder.
Just like you made a `Relation` folder, we can consider an `Access` or `Authorization` folder or `Provider` ?
:thinking:

Reply:
I am open to suggestions. I just put the classes there, because the old `ReadAccessFunction` (or similar, I don't remember) was in that place, too.

The `Relation` folder is created by `artisan` if you create the first relation. This was not my idea. But thanks for the credit. ;-)